Polyhedron/demo/Polyhedron/Plugins/PMP/Isotropic_remeshing_plugin.cpp

```diff
@@ -368,18 +369,36 @@ public Q_SLOTS:
         else
         {
           if (selection_item->selected_facets.empty() &&
-            (!selection_item->selected_edges.empty() || !selection_item->selected_vertices.empty()))
+              (!selection_item->selected_edges.empty() || !selection_item->selected_vertices.empty()))
+          {
+            if(protect &&
+               !CGAL::Polygon_mesh_processing::internal::constraints_are_short_enough(
+                 *selection_item->polyhedron(),
+                 selection_item->constrained_edges_pmap(),
+                 get(CGAL::vertex_point, *selection_item->polyhedron()),
+                 4. / 3. * target_length))
+            {
+              QApplication::restoreOverrideCursor();
+              QMessageBox::warning(mw, tr("Error"),
+                                   tr("Isotropic remeshing : protect_constraints cannot be set to"
+                                      " true with constraints larger than 4/3 * target_edge_length."
+                                      " Remeshing aborted."),
+                                   QMessageBox::Ok);
+              return;
+            }
+
             CGAL::Polygon_mesh_processing::isotropic_remeshing(
-              faces(*selection_item->polyhedron())
-              , target_length
-              , *selection_item->polyhedron()
-              , CGAL::Polygon_mesh_processing::parameters::number_of_iterations(nb_iter)
-              .protect_constraints(protect)
-              .edge_is_constrained_map(selection_item->constrained_edges_pmap())
-              .relax_constraints(smooth_features)
-              .number_of_relaxation_steps(nb_smooth)
-              .vertex_is_constrained_map(selection_item->constrained_vertices_pmap())
-              .face_patch_map(Patch_id_pmap<face_descriptor>()));
+                  faces(*selection_item->polyhedron())
+                  , target_length
+                  , *selection_item->polyhedron()
+                  , CGAL::Polygon_mesh_processing::parameters::number_of_iterations(nb_iter)
+                  .protect_constraints(protect)
+                  .edge_is_constrained_map(selection_item->constrained_edges_pmap())
+                  .relax_constraints(smooth_features)
+                  .number_of_relaxation_steps(nb_smooth)
+                  .vertex_is_constrained_map(selection_item->constrained_vertices_pmap())
+                  .face_patch_map(Patch_id_pmap<face_descriptor>()));
+          }
```

Comment:
please try to avoid to re-indent code like this. It makes a diff for nothing.
@maxGimeno can you please amend and "push force" this commit to cancel this re-indentation in the commit?
Thanks